code/modules/projectiles/gun.dm

```diff
@@ -455,48 +400,76 @@
 		else
 			//Do the same for the gun's unwielded spread
 			min_gun_sprd = round(spread_unwielded * wild_factor, 0.5)
-	bonus_spread = user.get_weapon_inaccuracy_modifier(target, src)
-	if(HAS_TRAIT(user, TRAIT_POOR_AIM)) //nice shootin' tex //Does not modify minimum spread, only maximum spread
-		bonus_spread += 25
+	var/bonus_spread = user.get_weapon_inaccuracy_modifier(target, src)
 	if(!is_wielded && requires_wielding)
 		bonus_spread += spread_unwielded
```

Comment:
This is applied twice, once in get_weapon_inaccuracy_modifier and once again here